codelab/03_buildpacks-runimage-override/tutorial.md

```diff
@@ -0,0 +1,86 @@
+# Using custom run image for CNB builder
+
+## Introduction
+
+### What is CNB?
+[Cloud Native Buildpacks](https://buildpacks.io/) (CNB) enable building
+a container image from source code without the need for a Dockerfile.
+Skaffold supports building with CNB, requiring only
+a local Docker daemon. 
+
+CNB uses two images when building an application image:
+  - A _run image_ serves as the base image for the resulting application image.
+  - A _build image_ acts as the host for performing the build.
+
+### What you'll learn
+
+- how to annotate an arbitrary image and use it as a CNB 
+[run image](https://buildpacks.io/docs/concepts/components/stack/).
+- how to integrate the custom run image with a sample Buildpacks application using Skaffold's *artifact dependencies* feature.
+
+___
+
+**Time to complete**: <walkthrough-tutorial-duration duration=10></walkthrough-tutorial-duration>
+
+Click the **Start** button to move to the next step.
+
+## First steps
+
+### Start a Minikube cluster
+
+We'll use `minikube` as our local kubernetes cluster of choice.
+
+Run:
+```bash
+minikube start
+```
+
+## Write a custom CNB run image
+
+CNB run and build images require some additional metadata to identify the _Stack ID_ and user/group accounts to be used.  A [_stack_](https://buildpacks.io/docs/concepts/components/stack/) is a specification or contract.  For example, the `io.buildpacks.stacks.bionic` stack defines that it provides the same packages as installed on Ubuntu 18.04.
+
+We add a base artifact with a single <walkthrough-editor-open-file filePath="base/Dockerfile">Dockerfile</walkthrough-editor-open-file> that defines the required metadata, and reference this as an <walkthrough-editor-select-line filePath="skaffold.yaml" startLine="6" startCharacterOffset="4" endLine="8" endCharacterOffset="0">artifact</walkthrough-editor-select-line> called `base` in our `skaffold.yaml`.
```

Comment:
```suggestion
We add a base artifact with a single <walkthrough-editor-open-file filePath="base/Dockerfile">Dockerfile</walkthrough-editor-open-file> that defines the required metadata, and reference this as an <walkthrough-editor-select-line filePath="skaffold.yaml" startLine="4" startCharacterOffset="4" endLine="6" endCharacterOffset="0">artifact</walkthrough-editor-select-line> called `base` in our `skaffold.yaml`.
```

I think this should be changed to this as when I clicked on it the `app` artifact was highlighted instead of `base`